src/components/portfolio/DetailsLineChart.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import Chart from 'react-apexcharts';
+
+type DetailsLineChartProps = {
+  portfolioValue: number[];
+};
+
+const DetailsLineChart: React.FC<DetailsLineChartProps> = ({
+  portfolioValue,
+}) => {
+  const [state] = useState({
```

Comment:
No state required here

Reply:
I deleted the component in its entirety since we will get a different one from the analyzer team anyways.